api/swagger.yml

```diff
@@ -681,6 +681,11 @@ components:
           type: string
         blockstore_namespace_ValidityRegex:
           type: string
+        warnings:
```

Comment:
This kind of conflicts with #2056 - I think we should fold both into a `/config/storage` endpoint and forgo the server-side warnings API (for now).